### ISSUE TITLE
gitolite configuration parsing support not compatible with python 2.6

### ISSUE THREAD

**Issue by iay** [git-multimail/git-multimail#134]
The following construct (at line 2908 as of 732940842cc6367833f547fa2232b8c62f3066b0) requires python 2.7 or 3.1 to work, and doesn't work under python 2.6:

```
re_template = r'^\s*#\s*{}\s*$'
```

The following does work (replace `{}` with `{0}`):

```
re_template = r'^\s*#\s*{0}\s*$'
```

**Comment by moy** [comment]
Note: you can click on the line number in GitHub to get a link to the right line. In this case: https://github.com/git-multimail/git-multimail/blob/732940842cc6367833f547fa2232b8c62f3066b0/git-multimail/git_multimail.py#L2908 

We do not cover this in the testsuite, which is bad (it would have been noticed immediately).

You fix works for Python 2.6, but not 2.4 which I try to support. It should be

    re_template = r'^\s*#\s*%s\s*$'

and then

    re.compile(re_template % x)

Can you send a pull-request with this?

**Comment by moy** [comment]
I'll add a test soon.

**Comment by moy** [comment]
Fixed in f9f329f131afb0c5338846.
